Assets/MRTK/Examples/Demos/LookAndPinch/Scenes/LookAndPinchDemo-02-TargetSelection.unity

```diff
@@ -0,0 +1,2751 @@
+%YAML 1.1
```

Comment:
Are these all basically duplicates of the existing ET demos but with a different pointer? Can we instead use the existing one but make it configurable? This feels like a lot of overhead if they're very similar demos and could lead to confusion (which I guess we could solve with documentation)

Reply:
There are plans to fully revamp the ConeCast demo scenes, but the intention for now I think is to provide a clear cut out of the box working implementation without requiring the users read the docs to get cone cast working.